ReactiveObjC/RACSignal+Operations.h

```diff
@@ -14,7 +14,7 @@
 @class RACMulticastConnection<__covariant ValueType>;
 @class RACScheduler;
 @class RACSequence<__covariant ValueType>;
-@class RACSubject;
+@class RACSubject<__covariant ValueType>;
```

Comment:
@erwald There's a pretty good writeup of this at https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)#Arrays